args/physics/PhysicsStatics.h

```diff
@@ -50,12 +50,12 @@ namespace args::physics
          * @return returns true if a seperating axis was found
          */
         static bool FindSeperatingAxisByExtremePointProjection(ConvexCollider* convexA
-            , ConvexCollider* convexB, const math::mat4& transformA, const math::mat4& transformB, HalfEdgeFace* refFace, float& maximumSeperation) 
+            , ConvexCollider* convexB, const math::mat4& transformA, const math::mat4& transformB, HalfEdgeFace** refFace, float& maximumSeperation) 
         {
             float currentMaximumSeperation = std::numeric_limits<float>::lowest();
- 
 
-            for (const auto face : convexB->GetHalfEdgeFaces())
+
+            for ( auto face : convexB->GetHalfEdgeFaces())
```

Comment:
this file should either be `physicsstatics.hpp` or `physics_statics.hpp` (I prefer the later for readability)